lib/elliptic/ec/index.js

```diff
@@ -164,3 +164,44 @@ EC.prototype.verify = function verify(msg, signature, key, enc) {
 
   return p.getX().mod(this.n).cmp(r) === 0;
 };
+
+EC.prototype.recoverPubKey = function(msg, signature, j) {
```

Comment:
Should do the `new Signature(...)` trick, and should have an encoding option for it.